### PR TITLE
Issue/2 fix selenium click on ubuntu

### PR DESCRIPTION
**Pull request by theoden-dd**
This PR adresses #2 .

The core commit is 16e2f4ef266bf8f6961421870b49827d948a8c7d .
I had success with it, but let me test it a bit more, I'll send a separate confirmation message later on.

In spite of that let's start the discussion right away.
Probably we need some changes in the readme file alongside etc.

### CODE REVIEW

**Comment by theoden-dd** [comment]
The tests were successful: met no more error reported.

File: .prpr-example.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Put this file in your home directory
+startrek_token: your_token_here
+
+# Stuff below is optional:
+free_work_owner: lepervushina
```
Comment (Roxe322) [inline .prpr-example.yaml]: let's change it please to robot-botlegger

File: prpr/download.py [hunk 2]
```diff
@@ -185,18 +186,22 @@ def _configure_firefox_driver(browser_settings, headless=False):
...
     page_load_timeout = PAGE_LOAD_TIMEOUT
     driver.maximize_window()
     driver.set_page_load_timeout(page_load_timeout)
     driver.get(revisor_url)
     try:
         element = driver.find_element_by_xpath(HISTORY_TAB_XPATH)
+        # The simple `element.click()` solution has stopped working on 19.05.2023.
+        # I dunno why, but this more complex approach is working.
+        action = ActionChains(driver)
+        action.move_to_element(element).click().perform()
```
Comment (Roxe322) [inline prpr/download.py]: What should we do with old click on 214 line?

https://github.com/Roxe322/prpr/pull/4/files#diff-26d6c853b0b43523a5214cb00a6b0148c12c9fb41575b0605ec973e13bc0db46R214

File: prpr/download.py [hunk 3]
```diff
@@ -185,18 +186,22 @@ def _configure_firefox_driver(browser_settings, headless=False):
...
 
 def _get_zip_urls(driver, revisor_url: str) -> list[str]:
+    logger.debug(f"Fetching from {revisor_url}...")
     page_load_timeout = PAGE_LOAD_TIMEOUT
     driver.maximize_window()
     driver.set_page_load_timeout(page_load_timeout)
     driver.get(revisor_url)
     try:
         element = driver.find_element_by_xpath(HISTORY_TAB_XPATH)
+        # The simple `element.click()` solution has stopped working on 19.05.2023.
```
Comment (Roxe322) [inline prpr/download.py]: Let's make this comment shorter or even remove it:

`simple element.click() doesn't work here for some reason`

File: .prpr-example.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# Put this file in your home directory
+startrek_token: your_token_here
+
```
Comment (Roxe322) [inline .prpr-example.yaml:3]: What's the matter of renaming this config? should we change something about setup in readme then?

**Review by Roxe322 — CHANGES_REQUESTED** [review]

**Review by Roxe322 — APPROVED** [review]

**Pull request merged**